PayloadPkg/OsLoader/LoadImage.c

```diff
@@ -857,6 +853,12 @@ LoadBootImages (
     LoadedImagesInfo->LoadedImageList[Index] = LoadedImage;
 
     if (EFI_ERROR (Status)) {
+      if (Index >= LoadImageTypeExtra0) {
+        // Continue boot if load extra image failed.
+        Status = EFI_SUCCESS;
+        continue;

```

Comment:
If Extra0 fails, do we need to load following Extra1/2/3..?

Reply:
there is no relation in Extra0/1/2/3. 
As long as Extra1 is configured in the boot option, current logic will load Extra1 regardless Extra0 load success or not.
But currently we don't have this use case.

Reply:
Understood. Thanks!